src/System.Management.Automation/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -249,6 +249,9 @@ private enum FormattingState
         /// </summary>
         private enum PreprocessingState { raw, processed, error }
 
+        const int defaultConsoleWidth = 120;
```

Comment:
This variable is only used in the method `GetConsoleWindowWidth`. Maybe it should be kept as a local const variable in that method.

I know both const vars here are with the value 120, but the naming suggests different things. `return defaultConsoleWidth`  in `GetConsoleWindowWidth` is more readable than `return StackAllocThreshold` in my opinion :)

Reply:
I put both constants next to each other specially because if we decide to change the first value, we probably want to change the second value too.

Reply:
I added the `private` modifier to 'DefaultConsoleWidth'.